sklearn/externals/setup.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
+import os
 
 
 def configuration(parent_package='', top_path=None):
     from numpy.distutils.misc_util import Configuration
     config = Configuration('externals', parent_package, top_path)
     config.add_subpackage('joblib')
+    config.add_subpackage('pilutil')
```

Comment:
we can move `_pilutil.py` to externals and name it `pilutils.py` and add `__all__` at the beginning of this file